### ISSUE TITLE
Missing variable assignment in processCreateBetterRollsMessage causes workflows to break

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#456]
In GitLab by @lucasmetzen on Jul 21, 2021, 04:32

# Unexpected behaviour (AKA bug)
When rolling an attack with midi-qol and BetterRolls activated, the chat message by BetterRolls is generated but the subsequent workflow(s) are not executed due to an error:

```
TypeError: Cannot read property 'token' of undefined
    at new Workflow (workflow.js:60)
    at new BetterRollsWorkflow (workflow.js:1892)
    at processCreateBetterRollsMessage (chatMesssageHandling.js:104)
```

When having Automated Animations activated, this prevents the weapon's animation to be displayed.

# Possible cause
In `processCreateBetterRollsMessage` on [line 43](https://gitlab.com/tposney/midi-qol/-/blob/master/src/module/chatMesssageHandling.ts#L43),
the variable `speaker` is declared but never assigned any value before being used as a parameter on 
[line 101](https://gitlab.com/tposney/midi-qol/-/blob/master/src/module/chatMesssageHandling.ts#L101):

```typescript
if (!workflow) workflow = new BetterRollsWorkflow(actor, item, speaker, targets, null);
```

# Suggested fix
Changing the mentioned line 43 to
```typescript
let speaker = message.data.speaker;
```
seems to fix the problem: no error message is shown in the console and the animation plays as expected.

**Comment by tposney** [comment]
In GitLab by @lucasmetzen on Jul 21, 2021, 04:33

PS: Thank you for having added support for BetterRolls! :heart:

**Comment by tposney** [comment]
Thanks for the heads up. I can't understand how that got overlooked in the code clean up. Fix will go out in 0.8.36
